configs/openeo-cloud.json

```diff
@@ -1,30 +1,147 @@
 {
   "index_name": "openeo-cloud",
   "start_urls": [
-    "https://docs.openeo.cloud"
+    {
+      "url": "https://open-eo.github.io/openeo-js-client/latest/",
+      "selectors_key": "js-client",
+      "page_rank": 0
+    },
+    {
+      "url": "https://open-eo.github.io/openeo-r-client/",
+      "selectors_key": "r-client",
+      "page_rank": 0
+    },
+    {
+      "url": "https://open-eo.github.io/openeo-python-client/",
+      "selectors_key": "python-client",
+      "page_rank": 0
```

Comment:
Default is 0 in case you want to clean those lines, but it's not necessary

Reply:
Thanks, I like it more if the order prio is directly clear so left them in. :-)